packages/rollup-plugin/src/index.ts

```diff
@@ -7,7 +7,7 @@ import {
   getSourceFromVirtualCssFile,
   virtualCssFileFilter,
 } from '@vanilla-extract/integration';
-import { relative, normalize, dirname } from 'path';
+import { relative, normalize, dirname } from 'path/posix';
```

Comment:
@riccardoperra This seems to have broken the test suite as it's not importable for some reason. Maybe  try the following instead which we've done elsewhere in vanilla-extract.

```ts
import { posix } from 'path';

const { relative, normalize, dirname } = posix;
```